main.cpp

```diff
@@ -0,0 +1,118 @@
+#include <iostream>
+
+class Sudoku {
```

Comment:
Add two files .h and .cpp for class Sudoku

Reply:
Resolved in 0993e81